### PR TITLE
Code refactory and UI improvements for ListDetails sample

### PR DESCRIPTION
**Pull request by ancirja-m**

### CODE REVIEW

File: ListDetail/src/androidTest/java/com/microsoft/device/display/samples/listdetail/ListDetailsInDualScreenModeTest.kt [hunk 1]
```diff
@@ -30,7 +30,7 @@ import org.junit.runner.RunWith
 @LargeTest
 class ListDetailsInDualScreenModeTest {
     @get:Rule
-    val activityRule = ActivityTestRule<MainActivity>(MainActivity::class.java)
+    val activityRule = ActivityTestRule<ListDetailsActivity>(ListDetailsActivity::class.java)
```
Comment (CesarValiente) [inline ListDetail/src/androidTest/java/com/microsoft/device/display/samples/listdetail/ListDetailsInDualScreenModeTest.kt:33]: Maybe we could take advantage of this refactor and instead `ActivityTestRule` that is already deprecated we start using `ActivityScenario` or `ActivityScenarioRule` --> https://developer.android.com/reference/androidx/test/rule/ActivityTestRule

Comment (CesarValiente) [reply ListDetail/src/androidTest/java/com/microsoft/device/display/samples/listdetail/ListDetailsInDualScreenModeTest.kt:33]: This is valid for all `ActivityTestRule` usage along this PR ;-)

Comment (ancirja-m) [reply ListDetail/src/androidTest/java/com/microsoft/device/display/samples/listdetail/ListDetailsInDualScreenModeTest.kt]: I added ActivityScenarioRule, but also I added the testOrchestrator library. Without testOrchestrator (https://developer.android.com/training/testing/junit-runner#using-android-test-orchestrator), the tests are failing, I don't know exactly why.

Comment (CesarValiente) [reply ListDetail/src/androidTest/java/com/microsoft/device/display/samples/listdetail/ListDetailsInDualScreenModeTest.kt]: Maybe is required when you want `ActivityScenarioRule` to behave exactly as `ActivityTestRule` forlaunching an initial Activity? (just guessing...)

Comment (ancirja-m) [reply ListDetail/src/androidTest/java/com/microsoft/device/display/samples/listdetail/ListDetailsInDualScreenModeTest.kt]: I tried to manually launch the activity with the ActivityScenarioRule, but this doesn't solve the problem.

**Review by bimiron — APPROVED** [review]

File: ListDetail/src/androidTest/java/com/microsoft/device/display/samples/listdetail/ListDetailsInDualScreenModeTest.kt (hunk 1 above)
Comment (CesarValiente) [reply ListDetail/src/androidTest/java/com/microsoft/device/display/samples/listdetail/ListDetailsInDualScreenModeTest.kt]: We agreed then that is totally fine to go with the testOrchestrator too ;-)

**Review by CesarValiente — APPROVED** [review]
LGTM! Thanks Andrei!

**Pull request merged**